requirements.txt

```diff
@@ -28,6 +28,7 @@ pytweening
 pywin32
 pyyaml
 rapidfuzz
+ruamel.yaml==0.18.6
```

Comment:
pyyaml converts the model to a JSON string and then uses json.loads to read the string. This will sort the keys alphabetically which is not what we want. Users are used to everything looking like this:

```
Uniques:
  - aspect: [ tibaults_will, 25 ]
    affix:
      - { name: damage_reduction_from_close_enemies, value: 12 }
```

With the default from pyyaml, it is instead printed like this:

```
Uniques:
-  affix:
      - { name: damage_reduction_from_close_enemies, value: 12 }
    aspect: [ tibaults_will, 25 ]
    ```

This still works just fine for filtering, but I think people will find it confusing and we'll get endless questions about it. It especially looks confusing when you have multiple uniques there, which is what happens after importing.